strings/strings.c

```diff
@@ -9,7 +9,17 @@
 */
 int string_length(char s[])
 {
-
+  int length = 0;
+  int i = 0;
+  //initialize both length and i at zero 
+  
+  
+  for(i = 0; s[i] != '\0'; i++){
+    length++;
+  } 
```

Comment:
`i` and `length` are basically the same thing. you could do this in a while loop or 
```
{
  int length = 0;
  for(; s[length] != '\0';){
    length++;
  } 
  return length;
} 
```